src/module.c

```diff
@@ -2725,12 +2725,13 @@ static int genericCallUnderscoreVariant(RedisModuleCtx *ctx, RedisModuleString *
   rm_asprintf(&localCmd, "_%.*s", len, cmd);
   /*
    * v - argv input array of RedisModuleString
+   * E - return errors as RedisModuleCallReply object (instead of NULL)
    * C - same client
    * M - respect OOM
    * 0 - same RESP protocol
    * ! - replicate the command if needed (allows for replication)
    */
-  RedisModuleCallReply *r = RedisModule_Call(ctx, localCmd, "vCM0!", argv + 1, argc - 1);
+  RedisModuleCallReply *r = RedisModule_Call(ctx, localCmd, "vECM0!", argv + 1, argc - 1);
```

Comment:
Should this be in a separate PR so it could be cherry-picked to other versions?

Reply:
Actually I can cherry-pick this to all the version branches, since I do not test the new `search` category we added to `8.0` only.
Added the backports.